dask_cuda/proxy_object.py

```diff
@@ -23,6 +23,13 @@
 except ImportError:
     from dask.dataframe.methods import concat_pandas
 
+try:
+    from dask.dataframe.dispatch import (
+        make_meta_dispatch as make_meta_dispatch,
+    )
```

Comment:
This may fit on one line. Would try running `black` again locally to see

Reply:
Yup, fixed it